Code/Editor/EditorViewportCamera.h

```diff
@@ -60,21 +60,22 @@ namespace SandboxEditor
     //! @param pitch Amount of pitch in radians.
     //! @param yaw Amount of yaw in radians.
     SANDBOX_API void InterpolateViewportCameraToTransform(
-        AzFramework::ViewportId viewportId, const AZ::Vector3& position, float pitch, float yaw);
+        AzFramework::ViewportId viewportId, const AZ::Vector3& position, float pitch, float yaw, float duration);
```

Comment:
Repeated: The comments above all these APIs should be updated to include the new parameter.

Reply:
Good call will add 👍